src/maliput_malidrive/builder/direction_usage_builder.h

```diff
@@ -92,13 +92,15 @@ class DirectionUsageBuilder {
   /// @param rule_id is the ID of the DirectionUsageRule.
   /// @param lane is Lane pointer.  It must not be nullptr and its type
   ///        must be `malidrive::Lane`.
+  /// @throws maliput::common::state_provider_error when `lane` is null.
```

Comment:
this should be rulebook_error.

State provider error should be reserved to phase and rule state providers.

Reply:
Done.